locked-asset/simple-lock-energy/src/unlock_with_penalty.rs

```diff
@@ -24,6 +26,10 @@ pub mod fees_collector_proxy {
         #[payable("*")]
         #[endpoint(depositSwapFees)]
         fn deposit_swap_fees(&self);
+
+        #[payable("*")]
+        #[endpoint(depositPenaltyFees)]
+        fn deposit_penalty_fees(&self);
```

Comment:
why is this function needed ? the same question for deposit_swap_fees ? this is simple lock energy contract - it does not get payments.

Reply:
Removed `deposit_penalty_fees`.